libs/env/src/env.ts

```diff
@@ -775,8 +775,9 @@ const envDefinitions = {
     parser: intParser,
     desc: "Configure the low battery percentage threshold",
   },
+  // TODO: TON switch to mainnet
   API_TON_ENDPOINT: {
```

Comment:
We could move this to `config.ts` like in near https://github.com/LedgerHQ/ledger-live/blob/e7ed028716bccb9dc29aa2fc672ecc7a3e78276a/libs/coin-modules/coin-near/src/config.ts

I think they recently added this way of adding env config variables.